swagger.yaml

```diff
@@ -114,7 +114,7 @@ paths:
                 runtime:
                   description: The runtime to use for the project
                   type: string
-                capability:
+                capabilities:
```

Comment:
No need to do it this time, but these are really two different PRs ;)

Reply:
@quintesse I've been lazy to make another PR 😇.. can we keep it ?

Reply:
Yeah of course :)